src/components/plots/areaPlot.ts

```diff
@@ -54,23 +54,19 @@ export module Plots {
       return new Plottable.Drawers.Area(key);
     }
 
-    protected _updateYDomainer() {
-      super._updateYDomainer();
-
+    protected _updateYScale() {
       var extents = this._propertyExtents.get("y0");
-      var extent = Utils.Methods.flatten(extents);
-      var uniqExtentVals = Utils.Methods.uniq(extent);
+      var extent = Utils.Methods.flatten<number>(extents);
+      var uniqExtentVals = Utils.Methods.uniq<number>(extent);
```

Comment:
It's not a cast, `uniq()` takes a generic. If I don't specify that it's operating on `<number>` it returns an `<any>`.